package.json

```diff
@@ -78,7 +78,8 @@
     "e2e": "./node_modules/.bin/codeceptjs run --steps",
     "wd:start": "node node_modules/protractor/bin/webdriver-manager update && node node_modules/protractor/bin/webdriver-manager start --standalone",
     "create-version": "npm --no-git-tag-version version",
-    "publish-version": "git tag $(node -e \"console.log(require('./package.json').version);\") && git push --tags"
+    "publish-version": "git tag $(node -e \"console.log(require('./package.json').version);\") && git push --tags",
+    "lint": "./node_modules/.bin/eslint ."
```

Comment:
Does this require `npm install eslint`? 
Right now we're using `grunt-eslint` which comes with its own version of `eslint`.
Should we add standalone `eslint` to `package.json`?

Reply:
grunt-eslint is dependent on eslint, so it gets installed in node_modules indirectly. I agree with you though, it's probably not good to be dependent on something without an explicit reference. Instead of this change I can add a new grunt command that calls lint without the other tests. I'll give that a go.

Reply:
@ynurmahomed I didn't realize ```grunt lint``` was a command. I'll just use that. I removed my changes to package.json. Thanks for pointing this all out.